sonic-chassisd/tests/mock_platform.py

```diff
@@ -52,10 +54,26 @@ def set_admin_state(self, up):
     def get_admin_state(self):
         return self.admin_state
 
+    def get_midplane_ip(self):
+        return self.midplane_ip
+
+    def set_midplane_ip(self):
+        if self.supervisor_slot == self.get_slot():
+            self.midplane_ip = '192.168.1.100'
+        else:
+            self.midplane_ip = '192.168.1.'+str(self.get_slot())
```

Comment:
Using `.format` or `%d` is better style than string concatenation in python.

Reply:
Fixed.